assignments/arrays.js

```diff
@@ -63,34 +63,62 @@ let inventory = [{"id":1,"car_make":"Lincoln","car_model":"Navigator","car_year"
 
 // ==== Challenge 1 ====
 // The dealer can't recall the information for a car with an id of 33 on his lot. Help the dealer find out which car has an id of 33 by logging the car's year, make, and model in the console log provided to you below:
-console.log(`Car 33 is a *car year goes here* *car make goes here* *car model goes here*` );
 
+//for loop, if inventory[i] = 32
+
+for (i = 0; i < inventory.length; i++) {
```

Comment:
use `for (let i = 0` not `for (i = 0`. The second one is an implicit assignment to global scope, which is not good.